vueapp/src/App.vue

```diff
@@ -133,6 +136,12 @@ export default {
 		passcodeLength: 4,
 		drawerShown: false,
 		store: store,
+		darkMode: false,
+		settings: {
```

Comment:
I don't see these `settings` being used anywhere. Seems like a small change and if it is present for something in the future, maybe we can add it in the future instead (it's good to keep the code as clean as possible and add related components/variables only when needed!)

Reply:
Ok yeah I'll remove the clunky components of the code then.